File: Python/anton_and_danik.py
n=int(input())
s=input()
x=0
y=0
for i in range(0,n):
    i=int(i)
    if s[i]=="A":
        x=x+1
    else:
        y=y+1
if x==y:
    print("Friendship")
elif x<y:
    print("Danik")
else:
    print("Anton")

File: Code blocks/anton_and_danik.c
#include<stdio.h>
int main()
{
    int n,i,j,x=0,y=0;
    char a[100002];
    scanf("%d",&n);
    scanf("%s",&a);
    for(i=0;i<n;i++)
    {
        if(a[i]=='A')
        {
            x++;
        }
        else if(a[i]=='D')
        {
            y++;
        }
    }
    if(x>y)
    {
        printf("Anton");
    }
    else if(x<y)
    {
        printf("Danik");
    }
    else
    {
        printf("Friendship");
    }
    return 0;
}
